docs/datascience/data-wrangler-quick-start.md

```diff
@@ -0,0 +1,87 @@
+# Quick Start Guide for Data Wrangler in VS Code
+
+[Data Wrangler](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler) is a code-centric data viewing and cleaning tool that is integrated into VS Code and VS Code Jupyter Notebooks. It provides a rich user interface to view and analyze your data, show insightful column statistics and visualizations, and automatically generate Pandas code as you clean and transform the data.
+
+![gif](https://github.com/microsoft/vscode-docs/assets/15910920/68d4e18f-dd99-4913-91b0-f36092e19a65)
```

Comment:
Update alt text and optionally add a paragraph of text Also review the other alt texts in the article.